tests/app/dao/test_invited_user_dao.py

```diff
@@ -25,7 +26,7 @@ def test_create_invited_user(notify_db_session, sample_service):
         "service": sample_service,
         "email_address": email_address,
         "from_user": invite_from,
-        "permissions": "send_messages,manage_service",
+        "permissions": "send_emails,manage_settings",
```

Comment:
This change seems kind of random, why are we changing the permissions in this way?

Reply:
Because the old permissions don't exist. The new way, the permissions absolutely need to exist in the permission enum, so I had to change them a little.

Reply:
Oooh, this is a good catch if the old permission didn't even exist in the first place; that could cause some other fun side effects that I'd think we'd rather not deal with!